lesson_11/test_homework_11.py

```diff
@@ -0,0 +1,70 @@
+import unittest
+from homework_11 import log_event
+
+username = "DorVit"
+status_1 = "successs"
+status_2 = "failed"
+status_3 = "expired"
```

Comment:
```suggestion
success = "success"
failed = "failed"
expired= "expired"
```
тут не згоден, success failed і expired набагато зрозуміліше загадкових статус_1